dftimewolf/lib/enhancers/timesketch.py

```diff
@@ -41,16 +41,17 @@ class TimesketchEnhancer(module.BaseModule):
   # Name given to all report containers.
   _REPORT_NAME = 'TimesketchEnhancer'
 
+  # For pytype
+  _formatter: utils.FormatterInterface
+  timesketch_api: "client.TimesketchApi"
+
   def __init__(self,
                state: "dftw_state.DFTimewolfState",
                name: Optional[str]=None,
                critical: bool=False):
     super(TimesketchEnhancer, self).__init__(
         state, name=name, critical=critical)
-    self.timesketch_api: "client.TimesketchApi"
-
     self._aggregations_to_skip = []  # type: List[str]
-    self._formatter: utils.FormatterInterface
     self._include_stories = False
```

Comment:
Lets be explicit and consistent and also specify these types, just like we're doing for types that are a List.
So include # type: bool etc.

Reply:
Done.